src/sentry/conf/server.py

```diff
@@ -1141,7 +1142,7 @@ def create_partitioned_queues(name):
 # The cache version affects both Django's internal cache (at runtime) as well
 # as Sentry's cache. This automatically overrides VERSION on the default
 # CACHES backend.
-CACHE_VERSION = 1
+CACHE_VERSION = 2 if six.PY3 else 1
```

Comment:
So this is actually `sys.version_info[0] - 1`? 😝

Reply:
lol, this took me a moment to get